plugin/src/test/groovy/com/github/triplet/gradle/play/PlayPublishTaskTest.groovy

```diff
@@ -273,68 +222,35 @@ class PlayPublishTaskTest {
         verify(editTracksMock, times(0)).update(anyString(), anyString(), eq('beta'), nullable(Track.class))
     }
 
-    @Test
-    void testApplicationIdChange() {
-        def project = TestHelper.evaluatableProject()
-
-        project.android {
-            flavorDimensions 'pricing'
-
-            productFlavors {
-                paid {
-                    dimension 'pricing'
-                    applicationId 'com.example.publisher'
-                }
-                free {
-                    dimension 'pricing'
-                }
-            }
-
-            buildTypes {
-                release {
-                    applicationIdSuffix '.release'
-                }
-            }
-
-            applicationVariants.all { variant ->
-                def flavorName = variant.variantData.variantConfiguration.flavorName
-                if (flavorName == 'paid') {
-                    variant.mergedFlavor.applicationId += '.paid'
-                }
-            }
-        }
-
-        project.evaluate()
-
-        // Attach the mock
-        setMockPublisher(project.tasks.publishApkPaidRelease)
-
-        // finally run the task we want to check
-        project.tasks.publishApkPaidRelease.publishApks()
-
-        // verify that we init the connection with the correct application id
-        verify(editsMock).insert('com.example.publisher.paid.release', null)
-    }
```

Comment:
Same here. Did this test break?

Reply:
Yup, see [above](https://github.com/Triple-T/gradle-play-publisher/pull/308#discussion_r193976041).